tests/test_json.py

```diff
@@ -121,6 +121,57 @@ class Squid(proto.Message):
     )
     assert json1 == '{"name":"Steve"}'
 
+    json1 = (
+        Squid.to_json(s, always_print_fields_with_no_presence=False)
+        .replace(" ", "")
+        .replace("\n", "")
+    )
+    assert json1 == '{"name":"Steve"}'
+
+    json1 = (
+        Squid.to_json(
+            s,
+            including_default_value_fields=False,
+            always_print_fields_with_no_presence=False,
+        )
+        .replace(" ", "")
+        .replace("\n", "")
+    )
+    assert json1 == '{"name":"Steve"}'
+
+    with pytest.raises(
+        ValueError,
+        match="Arguments.*always_print_fields_with_no_presence.*including_default_value_fields.*must match",
+    ):
+        Squid.to_json(
+            s,
+            including_default_value_fields=True,
+            always_print_fields_with_no_presence=False,
+        ).replace(" ", "").replace("\n", "")
+
+    with pytest.raises(
+        ValueError,
+        match="Arguments.*always_print_fields_with_no_presence.*including_default_value_fields.*must match",
+    ):
+        Squid.to_json(
+            s,
+            including_default_value_fields=False,
+            always_print_fields_with_no_presence=True,
+        ).replace(" ", "").replace("\n", "")
+
+    json2 = (
+        Squid.to_json(
+            s,
+            including_default_value_fields=True,
+            always_print_fields_with_no_presence=True,
+        )
+        .replace(" ", "")
+        .replace("\n", "")
+    )
+    assert (
+        json2 == '{"name":"Steve","massKg":0}' or json2 == '{"massKg":0,"name":"Steve"}'
+    )
```

Comment:
No action needed--just a comment. I was going to suggest we add test cases to ensure an empty `repeated` is printed only when the two parameters we're testing are true, but that gets into testing protobuf functionality. I think the other test cases you have and I suggest are enough to show we're presenting the right parameters to protobuf.